docs/user_guide.md

```diff
@@ -719,6 +720,76 @@ If you are a project admin, you can disable a Robot Account by clicking "Disable
+
+The action that is taken upon receiving a notification from a Harbor project depends on your continuous integration and development process. For example, by configuring Harbor to send a `POST` request to a webhook listener at an endpoint of your choice, you can trigger a build and deployment of an application whenever there is a change to an image in the repository.
+
+### Supported Events
+
+You can define one webhook per project. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL that you provide. The following table describes the events that trigger notifications and the contents of each notification.
+
+|Event|Webhook Event Type|Contents of Notification|
+|---|---|---|
+|Push manifest to registry|`IMAGE PUSH`|Repository namespace name, repository name, tags, manifest digest, image name, image size, push time timestamp, username of user who pushed image|
+|Pull manifest from registry|`IMAGE PULL`|Repository namespace name, repository name, manifest digest, image name, pull time timestamp, username of user who pulled image|
+|Delete manifest from registry|`IMAGE DELETE`|Repository namespace name, repository name, manifest digest, image name, image size, delete time timestamp, username of user who deleted image|
+|Upload Helm chart to registry|`CHART PUSH`|Repository name, chart name, chart type, chart version, chart size, tag, timestamp of push, username of user who uploaded chart|
+|Download Helm chart from registry|`CHART PULL`|Repository name, chart name, chart type, chart version, chart size, tag, timestamp of push, username of user who pulled chart|
+|Delete Helm chart from registry|`CHART DELETE`|Repository name, chart name, chart type, chart version, chart size, tag, timestamp of delete, username of user who deleted chart|
+|Image scan completed|`IMAGE SCAN COMPLETED`|Repository namespace name, repository name, tag scanned, image name, number of critical issues, number of major issues, number of minor issues, last scan status, scan completion time timestamp|
+|CVE found in image scan|`CVE FOUND`|Repository namespace name, repository repository, tags, vulnerability information (CVE ID, description, link to CVE, criticality, URL for any fix), username of user who performed scan|
+|Image scan failed|`IMAGE SCAN FAILED`|Repository namespace name, repository name, tag scanned, image name, error that occurred, username of user who performed scan|
+|Image replication|`IMAGE REPLICATION`|Repository name, replication type (docker image or helm chart), image/chart name, image/chart size, source registry type (Harbor or if 3rd party), destination registry type (Harbor or if 3rd party), source registry name, destination registry name, source namespace, destination namespace, authentication type, replication type (push or pull based) username of user who configured replication|

```

Comment:
This unfortunately didn't make it to the 1.10 release, so not supported

Reply:
Removed, and also removed the example use case for replication.